application/helpers/expressions/em_manager_helper.php

```diff
@@ -8548,8 +8550,17 @@ static function ProcessCurrentResponses()
                                     }
                                 }
                                 break;
-#                            case 'N': //NUMERICAL QUESTION TYPE
-#                            case 'K': //MULTIPLE NUMERICAL QUESTION
+                            case 'N': //NUMERICAL QUESTION TYPE
+                            case 'K': //MULTIPLE NUMERICAL QUESTION
+                                if (trim($value)!="") {
+                                    $aAttributes=$LEM->getQuestionAttributesForEM($LEM->sid, $qid, $_SESSION['LEMlang']);
+                                    if (!isset($aAttributes[$qid])) {
+                                        $aAttributes[$qid]=array();
+                                    }
+                                    if($aAttributes[$qid]['thousands_separator']=='1') {
+                                        $value=str_replace($invertRadix['separator'], '', $value);
+                                    }
+                                }
```

Comment:
@olleharstedt If you are agree with it, I can just update the file scripts/numerical_input.js to use the correct javascript part.